### DISCUSSION TITLE
Pronouns in profile

### DISCUSSION THREAD

**Discussion by gBasil** [modrinth/.github#187] [Feature requests]
<img width="340" alt="modmuss50's Modrinth profile" src="https://github.com/modrinth/.github/assets/57513430/a25600ec-38db-4a86-9a46-fa5aec223da9">

There could be a field for pronouns somewhere here.

**Comment by TheKodeToad** [comment]
+1 at first glance it doesn't seem useful because there are no comments, but it's useful for off-platform discussions. Edit: In fact maybe there will be comments! So it will be more useful in future!

PronounDB also used to work with Modrinth
![image](https://github.com/modrinth/.github/assets/57493648/a38e8b27-6290-4b6d-8863-559785eba8b8)

**Reply by AnOpenSauceDev** [reply]
Just wanted to add that [Modrinth will have comments in the future.](https://docs.modrinth.com/roadmap)

**Reply by TheKodeToad** [reply]
:+1: Edited
